config/software/inspec.rb

```diff
@@ -17,11 +17,15 @@
 name "inspec"
 default_version "master"
 
+license "Apache-2.0"
+license_file "LICENSE"
+
 source git: "https://github.com/chef/inspec.git"
 
 dependency "ruby"
 dependency "rubygems"
 dependency "bundler"
+dependency "nokogiri"
```

Comment:
Ok, it feels somewhat odd (especially installing test and integration deps during the build) but I assume there are reasons! :)